test/result_emitter_test.dart

```diff
@@ -1,20 +1,17 @@
 library result_emitter_test;
 
-import 'package:benchmark_harness/benchmark_harness.dart';
+import 'package:test/test.dart' show equals, expect, group, test;
```

Comment:
Unless there was some problem with conflicting names, remove the show clause.
Most users of the test package just import the whole namespace.